compiler/rustc_error_messages/locales/en-US/const_eval.ftl

```diff
@@ -4,4 +4,14 @@ const-eval-unstable-in-stable =
     .bypass-sugg = otherwise `#[rustc_allow_const_fn_unstable]` can be used to bypass stability checks
 
 const-eval-thread-local-access =
-    thread-local statics cannot be accessed at compile-time
\ No newline at end of file
+    thread-local statics cannot be accessed at compile-time
+
+const-eval-static-access =
+    { $kind -> 
+        [ConstFn] constant functions
```

Comment:
You could just return the "constant functions" string directly from `IntoDiagnosticArg`. I don't think it changes much for translators to pattern match on "ConstFn" to translate it than it would be to pattern match on "constant functions".